gcloud/monitoring/_helpers.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2015 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Helper functions for shared behavior."""
+
+
+def _format_timestamp(timestamp):
```

Comment:
I'm not sure what you mean.

This code accepts `datetime` objects with time zones. The serialization to Z-normalized RFC 3339 timestamps is as specified by the [Proto3 JSON Mapping](https://developers.google.com/protocol-buffers/docs/proto3#json).